kafka-plugins-0.8/pom.xml

```diff
@@ -51,6 +52,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
```

Comment:
Does this need to be 9.0.0-SNAPSHOT or 6.0.0-SNAPSHOT?

Reply:
fixed